rewrite_models.py

```diff
@@ -47,15 +49,24 @@ def rewrite_model(filename, inputs_dict, module_dict, attr_dict, forward_list):
             f.write(opstr.replace(",)\n", ")\n"))
         f.write("\n")
         f.write("m = M().eval()\n")
-        # f.write("ref_m = "+pretrained_models_str[modelname]+".eval()\n")
+        f.write("CORES=os.popen(\"lscpu | grep Core | awk '{print $4}'\").readlines()\n")
+        f.write("SOCKETS=os.popen(\"lscpu | grep Socket | awk '{print $2}'\").readlines()\n")
+        f.write("BS=int(CORES[0])*int(SOCKETS[0])\n")
+        f.write("batch_size=BS\n")
```

Comment:
BS is not always the number of cores. Other batch sizes are needed as well, e.g., 1, core_num/2, core_num, core_num*2, etc. Maybe we can add batch_size as a parameter and pass them from outside.

Reply:
have revert it, done later.